client/src/components/SimpleDataField.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const TextareaContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: ${(props) => props.theme.colors.input};
+  border-radius: 1.25rem;
+  border: none;
+  padding: 0.6rem;
```

Comment:
Just a personal opinion to add a bit more padding here:

```suggestion
  padding: 1rem;
```

Reply:
I like your personal opinion!